content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+### Comprehensive CONTRIBUTING.md
+
+<div class="release-feature-authors">authors: @alice-i-cecile</div>
+
+We now have a relatively complete [Contributors Guide](https://github.com/bevyengine/bevy/blob/main/CONTRIBUTING.md). If you are interested in contributing code or documentation to Bevy, that is a great place to start! 
+
+### CI Build System Improvements
+
+<div class="release-feature-authors">authors: @mockersf, @NathanSWard, @NiklasEi</div>
+
+We made a ton of CI improvements this release:
+
+* We now fail on cargo doc warnings
+* We now use [cargo deny](https://github.com/EmbarkStudios/cargo-deny) to protect against vulnerabilities, duplicate dependencies, and invalid licenses
+* PRs are now automatically labeled with the `S-Needs-Triage` label
+* Ci stability and speed improvements
+* We now check that our benchmarks build
+* We now assert compiler errors for compile_fail tests, giving us much stricter guarantees
+* Examples are now run using lavapipe (instead of swiftshader) for faster CI validation
+
+## What's Next For Bevy?
+
+Bevy development continues to pick up steam and we have no intention to slow down now!
+
+### The "Train" Release Schedule
+
+In the last two Bevy releases we made massive, sweeping changes to core systems. **Bevy 0.5** was "the one where we rewrote Bevy ECS". **Bevy 0.6** was "the one where we rewrote Bevy Render". These massive reworks took time, and as a result held back a bunch of other useful features and bug fixes.
+
+[The Bevy Community](https://bevyengine.org/community/) has reached relative consensus that we should have a more regular, more predictable release schedule. One where large features can't gum up the system.
+
+From now on, we will cut releases _approximately_ once every three months (as an upper bound ... sometimes we might release early if it makes sense). After the end of a release cycle, we will start preparing to cut a release. If there are small tweaks that need to be made or "life happens" ... we will happily postpone releases. But we won't hold releases back for "big ticket" items any more.
+
+We are balancing a lot of different concerns here:
+* Building trust with Bevy contributors that their changes will land in a timely manner
+* Building trust with Bevy users that they will receive regular updates and bug fixes
+* Giving enough time between releases to cut down on churn in the Bevy Plugin ecosystem (Bevy isn't "stable" yet, but longer releases give reasonable windows of "ecosystem stability") 
+* Providing enough content in a release to generate "hype". Bevy release blog posts tend to be a "rallying cry" for the community and I don't want to lose that.
+* Establishing proper work / life balance for core developers (crunch is bad!)
+    
+We will refine this process over time and see what works best.
+
+### More Renderer Features
```

Comment:
SSAO, compressed textures, more modular/reusable PBR shader types and functions, soft shadows, ... 😄 and then all the stuff we haven't even started on.

Reply:
Done (with one exception)! Can you elaborate on soft shadows? Does that fall under the current "shadow filters and cascades" category, or are you thinking more about "ray traced soft shadows"?

Reply:
No. Percentage Closer Soft Shadows.

Reply:
For now. :)